app/qml/pages/InfoPage.qml

```diff
@@ -62,7 +62,7 @@ Page {
             width: parent.width
             horizontalAlignment: Text.AlignHCenter
             wrapMode: Text.WrapAtWordBoundaryOrAnywhere
-            text: i18n.tr("This application let you synchronize with AsteroidOS smartwatches")
+            text: i18n.tr("This app lets you synch with AsteroidOS smartwatches")
```

Comment:
Any particular reason you used 'synch' instead of 'sync'? In your other commit you [are using 'sync'](https://github.com/AsteroidOS/telescope/pull/50/commits/936fea9e308d633cc5f88fab9a92589826cec297#diff-f30a6f85d7ee55488f1b3dd39197c118R103)
I'm not a native speaker so I might be wrong. The spelling just stood out to me (although that's technically a correct but less common spelling)

Reply:
I agree with what @jrtberlin said here. The best string here would be in my opinion: `This application lets you synchronize with AsteroidOS smartwatches`

Reply:
@jrtberlin Slip on my end. Good catch. Main point was to avoid "let" in error.

That being said…
@MagneFire The tally is 7 app/s to 2 application/s, this PR getting rid of both of the latter two.
https://hosted.weblate.org/languages/en/asteroidos/search/?offset=1&q=app&sort_by=-priority%2Cposition&checksum=

https://hosted.weblate.org/languages/en/asteroidos/search/?offset=1&q=sync&sort_by=-priority%2Cposition&checksum=    8-2 in favour of "sync".

Reply:
@comradekingu Thanks for your feedback. I agree that this stuff should be consistent. That being said. This is a description for the app. The usage of `app` is fine, but notice that on most sites it is referred to synchronization apps:
https://asteroidos.org/wiki/synchronization-clients/ (No mention of `sync`)
https://f-droid.org/en/packages/org.asteroidos.sync/
https://github.com/AsteroidOS/starfish

Also The app for Android is called `AsteroidOS Sync` this makes the tally actually: 4-3 for `sync`. I guess your change is fine because we have something similar in the Android client: `This app lets you sync your Android phone with your AsteroidOS watch`